app/models/rejection.rb

```diff
@@ -11,4 +11,12 @@ class Rejection < ActiveRecord::Base
   belongs_to :visit
 
   validates :reason, inclusion: { in: REASONS }
+
+  def pvo_possible?
```

Comment:
On the other hand, it's actually communicated in the interface as "VO" and "PVO", it's only relevant to prison staff, and it's just a flag/date that gets passed through and has no effect on anything else, so there's an argument that it's in the domain language.